infra/tf/configs/grafana/cloud.conf

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
```

Comment:
could we instead put these into a docker-compose just like we do with each node. Including the tracker, we just `dcoker-compose up`

Reply:
With this are you saying that grafana/prometheus/tracker should all be ran on a single machine with a docker compose?